### PR TITLE
Add SiFive InclusiveCache

### PR DESCRIPTION
**Pull request by zhemao**

### CODE REVIEW

File: generators/example/src/main/scala/Configs.scala [hunk 1]
```diff
@@ -247,3 +247,12 @@ class RV32BoomAndRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithRV32 ++
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new freechips.rocketchip.system.BaseConfig)
+
+class RocketL2Config extends Config(
```
Comment (abejgonzalez) [inline generators/example/src/main/scala/Configs.scala]: Can these be put in the BOOM, Rocket and (BOOM and Rocket) areas in the config file respectively? I think we should avoid having alot of configs spread out in a bunch of places.

Comment (colinschmidt) [reply generators/example/src/main/scala/Configs.scala]: Putting all rocket-chip configs in BOOM seems like its going to cause trouble eventually.

Comment (abejgonzalez) [reply generators/example/src/main/scala/Configs.scala]: Sorry if this was not clear. I mean just put these new configs in the sections related to boom/rocket/etc in this example config file.

Comment (abejgonzalez) [reply generators/example/src/main/scala/Configs.scala]: I pushed a fix.

File: generators/example/src/main/scala/Configs.scala [hunk 2]
```diff
@@ -247,3 +253,6 @@ class RV32BoomAndRocketConfig extends Config(
   new freechips.rocketchip.subsystem.WithRV32 ++
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
   new freechips.rocketchip.system.BaseConfig)
+
+class DualCoreRocketL2Config extends Config(
```
Comment (colinschmidt) [inline generators/example/src/main/scala/Configs.scala]: Can we also add a config that sets parameters in the L2 config?
`Rocket4BankL2` or `Rocket1MBL2` and just have them build in CI?
I think its important to test the number of banks and the number of memory channels. Having both of those parameters set has caused problems in the past and I think was the reason for Howie's rocket-chip PR.

Comment (abejgonzalez) [reply generators/example/src/main/scala/Configs.scala]: Sure. Thats fine with me.

**Comment by abejgonzalez** [comment]
Merging for now with knowing that BOOM is for some reason broken on this bump. Will address later.

**Comment by colinschmidt** [comment]
I think we should file issues about things that we will address later. I filed one about my L2 config complaint, and you should file one about fixing boom with the l2, IMO. Otherwise its very easy to lose track of things.

**Comment by davidbiancolin** [comment]
Issues or it never happened.

**Comment by abejgonzalez** [comment]
Thanks to @jerryz123 this: https://github.com/riscv-boom/riscv-boom/pull/315 should address the issue.

**Pull request merged**